### PR TITLE
Feature/conditions for sdl to send play tone param via speak request to hmi

### PR DESCRIPTION
**Pull request by dtrunov**
Removed PlayTone notification to HMI.
Moved this parameter to TTS.Speak.

### CODE REVIEW

File: src/components/application_manager/src/commands/mobile/alert_request.cc [hunk 1]
```diff
@@ -359,34 +360,22 @@ void AlertRequest::SendSpeakRequest(int32_t app_id) {
   using namespace smart_objects;
   // crate HMI speak request
   SmartObject msg_params = smart_objects::SmartObject(SmartType_Map);
-
-  msg_params[hmi_request::tts_chunks] =
-      smart_objects::SmartObject(SmartType_Array);
-  msg_params[hmi_request::tts_chunks] =
-    (*message_)[strings::msg_params][strings::tts_chunks];
+  if ((*message_)[strings::msg_params].keyExists(strings::tts_chunks) &&
```
Comment (AGaliuzov) [inline src/components/application_manager/src/commands/mobile/alert_request.cc]: You check the same value twice, maybe it worth to cache this value?

Comment (dtrunov) [inline src/components/application_manager/src/commands/mobile/alert_request.cc]: I applied your suggestion. Please make merging.

File: src/components/interfaces/HMI_API.xml [hunk 2]
```diff
@@ -2199,15 +2199,6 @@
         <description>Must be true if supported</description>
       </param>
     </function>
```
Comment (pestOO) [inline src/components/interfaces/HMI_API.xml]: xml version?

Comment (dtrunov) [inline src/components/interfaces/HMI_API.xml]: What did you mean?

**Pull request merged**